assets/components/ChatInput.svelte

```diff
@@ -25,6 +26,8 @@ $: inputParts = calculateInputParts(pos);
 $: nick = connection && connection.nick;
 $: placeholder = conversation.is('search') ? 'What are you looking for?' : connection && connection.is('unreachable') ? l('Connecting...') : l('What is on your mind %1?', nick);
 $: sendIcon = conversation.is('search') ? 'search' : 'paper-plane';
+$: pos && (conversation.update({input: inputEl.value}));
```

Comment:
No need for parenthesis around conversation.update().

Reply:
Fixed